File: function/5.py
in_str = input("이름을 입력해주세요.\n")

def login(_id):
    members = ['egoing', 'k8805']

    for member in members:
        if member == _id:
            return True
    return False        

if login(in_str):
    print('Hello, ' + in_str)
else:
    print('Who are you?')

File: function/5.rb
puts("이름을 입력해주세요.\n")
in_str = gets.chomp()

def login(id)
    members = ['egoing', 'k8805']

    for member in members do
        if member == id
            return true
        end
    end
    return false
end

if login(in_str)
    puts('Hello, ' + in_str)
else
    puts('Who are you?')
end